acceptance/python/src/test_divisible_asset.py

```diff
@@ -0,0 +1,176 @@
+# # Divisible assets integration testing
```

Comment:
If we ever generate documentation out of these tests, it would come handy to have the descriptions enclosed in 
 the triple quotes `"""` so that the rendering tool parses it properly.

Reply:
we do. You can test it by running `make doc-acceptance`